### PR TITLE
AbstractIOSelector pulled out close selector method

### PR DESCRIPTION
**Pull request by pveentjer**
Pulled out close selector method in the AbstractIOSelector.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java [hunk 1]
```diff
@@ -145,14 +145,19 @@ public final void run() {
...
+
+    private void closeSelector() {
+        if (logger.isFinestEnabled()) {
+            logger.finest("Closing selector " + getName());
+        }
+
+        try {
+            selector.close();
+        } catch (Exception e) {
+            Logger.getLogger(AbstractIOSelector.class).finest("Exception while closing selector", e);
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java]: you could use just `logger.finest()` instead?

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java]: Different pr :) Check the list I send you; it contains this fix.

Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java]: resolved

**Comment by mdogan** [comment]
needs rebase.

**Comment by pveentjer** [comment]
@mdogan rebased

**Comment by mdogan** [comment]
:+1:

**Pull request merged**